source/SOCVR.Chatbot/ChatbotActions/Commands/Permission/AddUserToGroup.cs

```diff
@@ -139,7 +146,7 @@ public override void RunAction(Message incomingChatMessage, Room chatRoom)
 
                 db.SaveChanges();
 
-                chatRoom.PostReplyOrThrow(incomingChatMessage, $"I've added @{chatTargetUser.Name.Replace(" ", "")} to the {requestingPermissionGroup} group.");
```

Comment:
The `.GetChatFriendlyUsername()` extension doesn't guarantee it'll return a "ping-valid" username. It should only be used when we want to refer to a user without pinging them.